lib/commands/url.js

```diff
@@ -1,7 +1,8 @@
 const { PERMISSIONS } = require("../core/Constants");
 
 module.exports = {
-    permissions: PERMISSIONS.Everyone,
```

Comment:
I guess you removed all `PERMISSIONS` usages, so you can probably remove it completely from `Constants.js` since it's useless now, it could be misleading.

Reply:
Yep, totally forgot.